central/imageintegration/datastore/datastore.go

```diff
@@ -42,12 +41,6 @@ func New(imageIntegrationStorage store.Store, indexer index.Indexer, searcher se
 		indexer:           indexer,
```

Comment:
Why does the datastore still use the indexer? Are you removing it in a separate PR?

Reply:
I'm going to back through and remove them